data/helpers.d/apt

```diff
@@ -332,7 +382,26 @@ ynh_add_app_dependencies () {
 # Requires YunoHost version 2.6.4 or higher.
 ynh_remove_app_dependencies () {
     local dep_app=${app//_/-}	# Replace all '_' by '-'
+
+    local current_dependencies=""
```

Comment:
This variable is not used? Could you remove the `if ynh_package_is_installed --package="${dep_app}-ynh-deps"`?